### ISSUE TITLE
Winget not asking for privilege escalation for some packages

### ISSUE THREAD

**Issue by FurtadoPires** [microsoft/winget-cli#1538]
# Brief description of your issue

Some applications like Libreoffice can't be installed with windows terminal in user mod, because the terminal doesn't ask to escalate privilege during the install process (crashing with the code 1603)

If it's executed using Powershell as admin, the install process silently finishes in the background but the terminal window remains stuck with the initiating install message.
# Steps to reproduce

`winget upgrade LibreOffice.LibreOffice`

# Environment

```none
[winget --info]
Windows: Windows.Desktop v10.0.19043.1237
Pacote: Microsoft.DesktopAppInstaller v1.16.12653.0

Registros: %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\DiagOutputDir

Links
-------------------------------------------------------------------------------
Política de Privacidade https://aka.ms/winget-privacy
Contrato de Licença     https://aka.ms/winget-license
Avisos de Terceiros     https://aka.ms/winget-3rdPartyNotice
Home page               https://aka.ms/winget
Termos da Windows Store https://www.microsoft.com/en-us/storedocs/terms-of-sale
Windows Package Manager version
Windows: Windows.Desktop version
Package: Microsoft.DesktopAppInstaller version

**Comment by denelon** [comment]
#1243 includes #1284 intended to address this scenario. We still have some work to do in the client for this, but the schema work has been implemented, and the support for the v1.1 manifest schema is in progress on the validation pipelines.

**Comment by denelon** [comment]
We've added the key to the schema to indicate when a package requires or prohibits elevation. This work item will be used to track the work to provide the information to the user. It will give an error for user mode if the package requires elevation, and it will give an error for admin mode if the package prohibits elevation.
